src/store/modules/auth.store.js

```diff
@@ -50,7 +50,61 @@ export const authStore = {
                         dispatch('alertStore/error', error, { root: true });
                     }
                 );
-        }     
+        },
+
+        chechUser( {dispatch, commit}, userId){
+            authService.chechUser(userId)
+            .then(
+                () => {
+                    if(userId)
```

Comment:
Why are you validating if `userId` after you have sent it to `authService` ? You should validate before and invoke it.

Reply:
done